src/tile/tile.scss

```diff
@@ -56,6 +53,20 @@
     }
   }
 
+  &[tabindex] {
```

Comment:
hm... Not sure any `tabindex` is enough.. There is -1 value, which should disable keyboard.

Reply:
I would prefer concrete class. 
Also, `tabindex=-1` is used to allow to focus element through JS but it is not focusable with keyboard. We are using this to make arrow navigation to work.

Reply:
Using `.iui-actionable`.